org/serializers.py

```diff
@@ -89,3 +89,8 @@ def save(self):
 
         return [org,admin_group,admin_permission_set]
 
+
+class EditOrgSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Org
+        fields = ('name','tagline','about')
```

Comment:
input checking need to be performed. when I dont provide a specific field then it shoots out error 500.

![image](https://user-images.githubusercontent.com/33046846/82859927-c0813e00-9f35-11ea-9059-f0386251a786.png)
![image](https://user-images.githubusercontent.com/33046846/82859928-c545f200-9f35-11ea-9017-6a2a845be3a6.png)